app/components/Icons/SidebarLeftButton.tsx

```diff
@@ -0,0 +1,28 @@
+const SidebarLeftButton = () => {
+  return (
+    <svg
+      width="32"
+      height="32"
+      viewBox="0 0 32 32"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <rect width="32" height="32" rx="4" fill="#1B242F" />
+      <g clip-path="url(#clip0_552_8662)">
+        <path
+          d="M18.4999 9.33325V22.6666M14.3333 14.3333L12.6666 15.9999L14.3333 17.6666M10.9999 9.33325H20.9999C21.9204 9.33325 22.6666 10.0794 22.6666 10.9999V20.9999C22.6666 21.9204 21.9204 22.6666 20.9999 22.6666H10.9999C10.0794 22.6666 9.33325 21.9204 9.33325 20.9999V10.9999C9.33325 10.0794 10.0794 9.33325 10.9999 9.33325Z"
+          stroke="#FAFAFA"
+          stroke-linecap="round"
+          stroke-linejoin="round"
```

Comment:
It's better if we change this properties to avoid warnings
clip-path -> clipPath
stroke-linecap -> strokeLinecap
stroke-linejoin -> strokeLinejoin